controllers/front/creditCards.php

```diff
@@ -82,10 +83,10 @@ public function postProcess()
         if ($selectedCard) {
             $cardAlias = new SaferPayCardAlias($selectedCard);
             if ($cardAlias->delete()) {
-                $this->success[] = $this->l('Successfully removed credit card');
+                $this->success[] = $this->module->l('Successfully removed credit card');
```

Comment:
self::FILENAME you forgot here I assume. These strings will not be translatable without it in this file